pygeodiff/main.py

```diff
@@ -18,11 +18,35 @@ class GeoDiff:
     def __init__(self, libname=None):
         """
         if libname is None, it tries to import c-extension from wheel
-        messages are shown in stdout/stderr.
+        messages are shown in stdout/stderr. C-Library and context is lazy-loaded.
+
         Use environment variable GEODIFF_LOGGER_LEVEL 0(Nothing)-4(Debug) to
         set level (Errors by default)
         """
-        self.clib = GeoDiffLib(libname)
+        self.libname = libname
+        self.clib = None
```

Comment:
@wonder-sk should this clib be shared between all GeoDiff() instances? (on module level?)